README.md

```diff
@@ -70,9 +70,9 @@ One of the easiest ways to contribute is to participate in discussions. There ar
 
 | Item        | Value  |
 |---------------------|---|
-| Mailing List | Coming soon! |
-| Slack Channel       | Coming soon! |
-| Community meeting | Every other Tuesday at 10:30AM PST. Zoom meeting link: https://zoom.us/j/623691799?pwd=ZWc4SHFNdWpRUVVNYkdJWE9zVHpjZz09  |
-| Meeting Notes       | https://docs.google.com/document/d/1nqdFEyULekyksFHtFvgvFAYE-0AMHKoS3RMnaKsarjs/edit?usp=sharing |
+| Mailing List | https://groups.google.com/forum/#!forum/oam-dev |
+| Meeting Information | Bi-weekly (Starting October 23rd), Tuesdays 10:30AM PST  |
+| Meeting Link | https://zoom.us/j/271516061  |
+| IM Channel       | https://gitter.im/oam-dev/ |
```

Comment:
why do we remove the meeting notes doc?

Reply:
See #192

Reply:
I added it back in, I must have left it out by accident